picky/src/pkcs12/safe_bag.rs

```diff
@@ -20,7 +20,7 @@ pub struct SafeBag {
 }
 
 impl SafeBag {
-    /// Create new safe bag with private key
+    /// Create new safe bag holding a private key
     pub fn new_key(key: PrivateKey, attributes: Vec<Pkcs12Attribute>) -> Result<Self, Pkcs12Error> {
```

Comment:
Maybe we could remove `attributes` from `new_key` after we added `add_attribute`. WDYT?

Reply:
I wasn’t too sure because `add_attribute` is less efficient (recomputing the inner value with each call), but I can do that yes!